Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class CaesarCipher
+    {
+        public static void Demo()
+        {
+            bool isContinue = true;
+            while (isContinue)
+            {
+                int shift = UserSelectSizeAndDirectionOfShift();
+                string forOperation = RequestInput();
+                ConvertString(shift, forOperation);
+                isContinue = askIfRepeating();
+            }
+        }
+      
+            private static bool askIfRepeating()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'askIfRepeating' to match pascal case naming rules, consider using 'AskIfRepeating'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5002934)